### ISSUE TITLE
Add New Output To Export All EBD In One Excel File

### ISSUE THREAD

**Issue by hf-krechan** [Hochfrequenz/ebdamame#217]
Claudia would like to have an Excel file which contains all EBD tables.

The goal is to have something similar like the following excel file
[EBD_GPKE_LFW.xlsx](https://github.com/user-attachments/files/16130655/EBD_GPKE_LFW.xlsx)

We do not need to fill column `Häufigkeit`. 

## Features

- [ ] Add a new subcommand to the ebdamame cli interface to export one excel file with all EBDs. See [kohlrahbi as an example](https://github.com/Hochfrequenz/kohlrahbi/blob/5f9c49211f07687fe55b6c38e42b6089fa43691d/src/kohlrahbi/__init__.py#L20-L22).
- [ ] Add flag to choose `format version`
- [ ] Add flag to choose `EBDs` (if no EBD flag is given, export all EBDs)
- [ ] Add flag to set `path to edi energy mirror` (there are all EBD documents)
- [ ] Add flag to set `path to machine readable anwendungshandbücher` (this is needed to check in which prüfis is the EBD)

**Comment by hf-kklein** [comment]
OK, let's split this into easy-to-handle isolated features:

- [ ] (0) allow excel export
- [ ] (1) instead of a docx file it should be possible to provide a path to a directory
- [ ] (2) add a wrapper around (1) that, given a path to edi_energy_mirror, exports all the files from a specific format version (which translates to a sub-directory)
- [ ] (3) add a feature that, given a list of machine-readable AHBs, returns the prüfis of those that refer to a given EBD (str)
- [ ] (4) add a wrapper around (3) that accepts the machine-readable-ahb repo + format version as argument, and reads the AHB information from there

So basically it's 3 real features (0), (1) and (3) and two utiliy functions/wrappers (2) + (4) around those features.
